org.eclipse.jdt.ls.debug/src/org/eclipse/jdt/ls/debug/adapter/StackFrameScope.java

```diff
@@ -0,0 +1,40 @@
+/*******************************************************************************
+* Copyright (c) 2017 Microsoft Corporation and others.
+* All rights reserved. This program and the accompanying materials
+* are made available under the terms of the Eclipse Public License v1.0
+* which accompanies this distribution, and is available at
+* http://www.eclipse.org/legal/epl-v10.html
+*
+* Contributors:
+*     Microsoft Corporation - initial API and implementation
+*******************************************************************************/
+
+package org.eclipse.jdt.ls.debug.adapter;
+
+import java.util.Objects;
+
+import com.sun.jdi.StackFrame;
+
+public class StackFrameScope {
```

Comment:
I don't see it used anywhere. Please include in other change lists that really use it.